Vinyl/Response.swift

```diff
@@ -60,9 +60,9 @@ func ==(lhs: Response, rhs: Response) -> Bool {
 extension Response: Hashable {
     
     var hashValue: Int {        
-        let body = self.body == nil ? "\(self.body)" : ""
-        let error = self.error == nil ? "\(self.error)" : ""
+        let body = self.body == nil ? "" : "\(self.body!)"
```

Comment:
No, because `self.body` is a `Data` and `""` is a `String`, and we want the result to be a `String`. In the case of the body this compiles:

`let body = "\(self.body ?? Data())"`
But the equivalent for the Error does not...